src/main/java/org/gridsuite/shortcircuit/server/utils/SpecificationUtils.java

```diff
@@ -34,15 +34,15 @@ private SpecificationUtils() {
 
     // we use .as(String.class) to be able to works on enum fields
     public static <X> Specification<X> equals(String field, String value) {
-        return (root, cq, cb) -> cb.equal(getColumnPath(root, field).as(String.class), value);
+        return (root, cq, cb) -> cb.equal(cb.upper(getColumnPath(root, field)).as(String.class), value.toUpperCase());
     }
 
     public static <X> Specification<X> contains(String field, String value) {
-        return (root, cq, cb) -> cb.like(getColumnPath(root, field), "%" + EscapeCharacter.DEFAULT.escape(value) + "%", EscapeCharacter.DEFAULT.getEscapeCharacter());
+        return (root, cq, cb) -> cb.like(cb.upper(getColumnPath(root, field)), "%" + EscapeCharacter.DEFAULT.escape(value).toUpperCase() + "%", EscapeCharacter.DEFAULT.getEscapeCharacter());
```

Comment:
The value can't be null?
intellij suggest to add : Objects.requireNonNull(EscapeCharacter.DEFAULT.escape(value)).toUpperCase()

Reply:
If `value` of filter is null it will broke before during the `toString()`.
I didn't manage it but maybe I should use NonNull on every params of the record ?

Reply:
Actually I use null in some cases (for contains by instance, but I will probably change later).
I add it in my TODO list to check when I come back.